src/OSPSuite.Core/Domain/Builder/ExpressionParameter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class ExpressionParameter : PathAndValueEntity, IStartValue
+   {
+      /// <summary>
+      ///    Do not use! When refactoring on promotion to core, this should be removed
+      /// </summary>
+      public double? StartValue { get; set; }
```

Comment:
Should this be removed?

Reply:
The very next issue after this PR is going to be refactoring all the StartValues -> EntityWithPath to make sure it makes sense.

I had this code with this comment at one point, but then we removed the PathWithEntity.Value and just had StartValue to eliminate confusion.

For this PR, I would not bother to take the comment out.

Reply:
ok